pubsub/gcloud/aio/pubsub/subscriber.py

```diff
@@ -99,6 +100,20 @@ async def acker(subscription: str,
                 log.warning(
                     'Ack request failed, better luck next batch', exc_info=e)
                 metrics_client.increment('pubsub.acker.batch.failed')
+
+                is_response_error = isinstance(
+                    e,
+                    aiohttp.client_exceptions.ClientResponseError)
+                if is_response_error and e.status == 400:  # type: ignore # pylint: disable=no-member
+                    log.warning(
```

Comment:
This probably warrants an `error` level notification IMO -- this feels like the sort of thing which folks should be loudly told about.

Reply:
Hmm, yes, that makes sense. It is kinda actionable meaning that it won't be triggered by random network blip, but rathe an actual bug or unfortunate but otherwise valid infrastructure even (like in our case). 👍